ansible/roles/test/files/helpers/change_mac.sh

```diff
@@ -7,7 +7,7 @@ INTF_LIST=$(ls /sys/class/net | grep -E "^eth[0-9]+$")
 for INTF in ${INTF_LIST}; do
     ADDR="$(cat /sys/class/net/${INTF}/address)"
     PREFIX="$(cut -c1-15 <<< ${ADDR})"
-    SUFFIX="$(printf "%02x" ${INTF##eth})"
+    SUFFIX="$(printf "%02x" $((${INTF##eth}%256)))"
```

Comment:
I think we will have duplicated intf MAC once it's over 256?  
Should we change it to something like the following?
```
    PREFIX="$(cut -c1-13 <<< ${ADDR})"
    INTF_ID=${INTF##eth}
    SUFFIX="$(printf "%x:%02x" $(expr ${INTF_ID} / 256) $(expr ${INTF_ID} % 256))"
```

Reply:
Ah I see, you cut just two additional characters from the prefix, so yes your suggestion should work as-is.

Reply:
Suggestion implemented in latest commit.